test/should_fail/record_exhaustive.erl

```diff
@@ -0,0 +1,69 @@
+-module(record_exhaustive).
+
+-compile(export_all).
```

Comment:
```suggestion
-compile([export_all, nowarn_export_all]).
```

Reply:
I've been trying to remove as many warnings from the test suite as possible lately.

Reply:
done